Add icon for custom integration ABB Free@Home

kingsleyadam opened the pull request:
<!--
  You are amazing! Thanks for contributing to our project!
  Please, DO NOT DELETE ANY TEXT from this template! (unless instructed).
-->
## Proposed change
<!-- 
  Describe the big picture of your changes here to communicate to the
  maintainers why we should accept this pull request.
-->

This adds new icon files for ABB Free@Home Custom Integration.


## Type of change
<!--
  What type of change does your PR introduce to the Home Assistant Brands?
  NOTE: Please, check only 1! box! 
  If your PR requires multiple boxes to be checked, you'll most likely need to
  split it into multiple PRs. This makes things easier and faster to code review.
-->

- [ ] Add a new logo or icon for a new core integration
- [ ] Add a missing icon or logo for an existing core integration
- [X] Add a new logo or icon for a custom integration (custom component)
  - [X] I've added a link to my custom integration repository in the PR description
  - [ ] I've opened up a PR for my custom integration on the [Home Assistant
    Python wheels repository](https://github.com/home-assistant/wheels-custom-integrations)
- [ ] Replace an existing icon or logo with a higher quality version
- [ ] Replace an existing icon or logo after a branding change
- [ ] Removing an icon or logo

## Additional information
<!--
  Details are important, and help maintainers processing your PR.
  Please be sure to fill out additional details, if applicable.
-->

- This PR fixes or closes issue: fixes #
- Link to code base pull request: 
- Link to documentation pull request: 
- Link to integration documentation on our website: 
- Link to custom integration repository: https://github.com/kingsleyadam/local-abbfreeathome-hass

## Checklist
<!--
  Put an `x` in the boxes that apply. You can also fill these out after
  creating the PR. If you're unsure about any of them, don't hesitate to ask.
  We're here to help! This is simply a reminder of what we are going to look
  for before merging your contribution.
-->

- [X] The added/replaced image(s) are **PNG**
- [X] Icon image size is 256x256px (`icon.png`)
- [X] hDPI icon image size is 512x512px for  (`icon@2x.png`)
- [ ] Logo image size has min 128px, but max 256px, on the shortest side (`logo.png`)
- [ ] hDPI logo image size has min 256px, but max 512px, on the shortest side (`logo@2x.png`)

<!--
  Thank you for contributing <3
-->

frenck reviewed (changes requested):
Hi there @kingsleyadam 👋 

Thanks for the PR, I do have some things that need to be addressed before this PR can be merged.

The provided branding, is not the branding of ABB, it looks like some kind of sub or product specific branding:

![CleanShot 2024-10-07 at 09 04 46@2x](https://github.com/user-attachments/assets/026400f3-7f7f-4831-9439-235b6e9dec61)

Please use the ABB branding instead.

Additionally, the integration domain provided is `abbfreeathome-ci`, however, the use of a dash `-` in an integration domain isn't valid. Suggestion is to replace it with an underscore `_` instead.

../Frenck

kingsleyadam commented:
> Hi there @kingsleyadam 👋
> 
> Thanks for the PR, I do have some things that need to be addressed before this PR can be merged.
> 
> The provided branding, is not the branding of ABB, it looks like some kind of sub or product specific branding:
> 
> ![CleanShot 2024-10-07 at 09 04 46@2x](https://private-user-images.githubusercontent.com/195327/374065070-026400f3-7f7f-4831-9439-235b6e9dec61.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjgyODg5NTcsIm5iZiI6MTcyODI4ODY1NywicGF0aCI6Ii8xOTUzMjcvMzc0MDY1MDcwLTAyNjQwMGYzLTdmN2YtNDgzMS05NDM5LTIzNWI2ZTlkZWM2MS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDA3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAwN1QwODEwNTdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04NGJjMDkwZTJmMDMxYWY2YjNlZWQ0ZjEzMTE5MGUxZGE1NWQzZDlhYTA3NzA0MTg5MmQ1ZjM1MzU5OWNmOTYyJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.7nD2Hfm7jDsfZEUVajrgGg1zRSien2I0DkHQQ4ztaP8)
> 
> Please use the ABB branding instead.
> 
> Additionally, the integration domain provided is `abbfreeathome-ci`, however, the use of a dash `-` in an integration domain isn't valid. Suggestion is to replace it with an underscore `_` instead.
> 
> ../Frenck

Thanks for the freedback @frenck! I used the free@home branding since ABB is such a big company with many brands. I'm also good with just using ABB. I've updated the PR to be ABB branding.

I've also updated the domain to use an underscore vs a dash both in this PR and my custom integrations repo.

frenck reviewed (approved):
Thanks, @kingsleyadam 👍 

../Frenck
